### DISCUSSION TITLE
Idea: Hide macros in the context menu

### DISCUSSION THREAD

**Discussion by samaguire** [TabularEditor/TabularEditor3#1091] [Ideas]
As per the title. Can we have a new option to hide a macro in the context menu? i.e. like the existing disable macro option.

i.e. the below macro is only needed in a custom toolbar, given it can only successfully run when editing a DAX query, so it would be great to not have it show in the macro context menu.

NB: disabling the macro disables it in the custom toolbar so isn't an option in this scenario

![image](https://user-images.githubusercontent.com/62320770/155395675-da7888ce-b80b-4630-8f22-6d6f44ea58bf.png)

![image](https://user-images.githubusercontent.com/62320770/155396412-6d74bd30-9d45-4f87-abd2-95defc6f0960.png)

**Comment by otykier** [comment]
Good suggestion. Thanks!
